Source/BugsnagClient.m

```diff
@@ -42,12 +42,6 @@
 #import "BSG_KSSystemInfo.h"
 #import "BSG_KSMach.h"
 
-#if TARGET_IPHONE_SIMULATOR || TARGET_OS_IPHONE
```

Comment:
Just wanted to check why it's necessary to move these imports to the header file?

Reply:
Good spot.  Hangover from trying to make `orientationChanged` testable.  Moved back.